backend/lib/endpoints/feedback.js

```diff
@@ -0,0 +1,44 @@
+const { createFeedbackSchema } = require("./schema/feedback");
+
+/*
+ * /api/feedback
+ */
+async function routes(app) {
+  const Feedback = app.mongo.model("Feedback");
+
+  app.post("/", { schema: createFeedbackSchema }, async (req, reply) => {
+    const { userId } = req.body;
+    let ip = "undefined";
+    if (userId) {
+      const userFeedback = await Feedback.find({ userId });
+      if (userFeedback[0]) {
+        reply.conflict("User already submitted feedback");
```

Comment:
yeah so this is not a good idea because the code is not returning here inside the if, so it's really not clear what happens below this block of code... if that code will be executed or not. 
This kind of if block would fit better with "early return" and I would still prefer the built-in http errors combined with the return for consistency (and like explained in the comment about fastify best practices with async/await).

Reply:
I do like that this is using the shorter `reply.[httpError]` syntax so maybe we can use this everywhere but maybe use `throw` since it's an error and should not continue on the next line?